plugins/statistics.koplugin/main.lua

```diff
@@ -2678,12 +2736,12 @@ function ReaderStatistics:onPageUpdate(pageno)
         return
     end
 
+    local closing = false
     -- We only care about *actual* page turns ;)
     if self.curr_page == pageno then
         return
     end
 
-    local closing = false
```

Comment:
Thought I already mentionned that and you fixed it - but let this where it was (the red location is right).